### DISCUSSION TITLE
Datetime issue

### DISCUSSION THREAD

**Discussion by josephmancuso** [MasoniteFramework/orm#470] [General]
I haven't used this discussion feature before but maybe we can discuss this datetime issue here? 

@erhuabushuo Your change didn't look right to me last night ..

This is the part that was confusing to me. You are checking here is the `_datetime` is an instance of the `datetime` module right? isn't this wrong? Shouldn't this be checking either `datetime.time` or `datetime.date` or `datetime.datetime`, etc?

```python
elif isinstance(_datetime, datetime):            
    return pendulum.instance(_datetime, tz=self.__timezone__)
```

**Comment by erhuabushuo** [comment]
```
>>> import datetime
>>> issubclass(datetime.datetime, datetime.date)
True
```

this datetime is not module , we import this by `from datetime import datetime`,  it is datetime.datetime 

datetime is `subclass` of date, so` isinstance(datetime, date)` will aslo be `True`,   if the database raw type is `datetime`,   we put isintance(_datetime, date) before isinstance(_datetime, datetime),  It will be cast to  date type here by `return pendulum.datetime( _datetime.year, _datetime.month, _datetime.day, tz=self.__timezone__)`. the last code `return pendulum.instance(_datetime, tz=self.__timezone__)` would not be execute.

**Comment by josephmancuso** [comment]
After looking into this more and actually writing a test around this, you are correct. I have reverted my change. Thanks so much for taking the time to look into this
